src/components/Player.ts

```diff
@@ -0,0 +1,102 @@
+import { GameObj, KaboomCtx } from "kaplay";
+import GameUtils from "../game/GameUtils";
+import GameInstance from "../game/Game";
+
+export default class Player { 
+    public static playerObject: GameObj;
+    private static game: KaboomCtx = GameInstance.getGameInstance().getKObject(); 
+
+    private static playerComponents = [
```

Comment:
that could be easily made by passing options to the `createGameObject` in the GameObject class